src/main/scala/com/netflix/edda/mongo/MongoDatastore.scala

```diff
@@ -147,7 +160,6 @@ object MongoDatastore {
         val queryTimeout = Utils.getProperty("edda.collection", "queryTimeout", name, "60000").get.toInt
 
         val options = new MongoOptions
-        options.autoConnectRetry = true
```

Comment:
It's deprecated.  I was hoping to get Edda to a 3.2 or 3.3 mongo driver.  2.13 is the intermediate step to upgrade and will start issuing warnings about other deprecated API calls during the build so they can be fixed.

http://mongodb.github.io/mongo-java-driver/3.0/whats-new/upgrading/